modules/gitops-enabling-the-application-set-resources-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * declarative_clusterconfig/managing-app-sets-in-non-control-plane-namespaces.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-enabling-the-application-set-resources-in-non-control-plane-namespaces_{context}"]
+= Enabling the application set resources in non-control plane namespaces
+
+As an Argo CD administrator, you can define a certain set of non-control plane namespaces wherein you can create, update, and reconcile `applicationSet` resources. You must explicitly enable and configure the Argo CD and `applicationSet` custom resources (CRs) as per your requirements.
+
+.Procedure
+
+. Set the `sourceNamespaces` parameter for the `applicationSet` spec to include the non-control plane namespaces:
++
+.Example Argo CD custom resource
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: repo
```

Comment:
```suggestion
```
We can remove this label as it doesn't add any significant value to the example.